libs/storage/src/entities.rs

```diff
@@ -19,3 +23,38 @@ impl CreateCollabParams {
     }
   }
 }
+
+#[derive(Debug, Clone, sqlx::FromRow)]
+pub struct AfWorkspace {
+  pub workspace_id: uuid::Uuid,
+  pub database_storage_id: Option<sqlx::types::uuid::Uuid>,
+  pub owner_uid: Option<i64>,
+  pub created_at: Option<DateTime<Utc>>,
+  pub workspace_type: i32,
+  pub deleted_at: Option<DateTime<Utc>>,
+  pub workspace_name: Option<String>,
+}
+
+#[derive(Debug, sqlx::FromRow)]
+pub struct AfUserProfileView {
```

Comment:
Would it be better if replacing the AF instead of Af? Maybe we can document it and stick to that style.

Reply:
yeah we can do that, will make the changes